torch/_higher_order_ops/associative_scan.py

```diff
@@ -63,8 +92,15 @@ def associative_scan(
             This function must be pure, pointwise, and satisfy the associative property.
```

Comment:
The comment in line 86 above is stale, can you update it?

Reply:
Done